lib/e_chronicler_web/templates/journal_entry/show.html.heex

```diff
@@ -3,10 +3,17 @@
         <h3 id="title"> <%= @entry.title %></h3>
         <div id="metadata">
         Author: <%= @entry.author %> <br>
-        Published: <%= @entry |> EChronicler.Models.JournalEntry.format_datetime() %> 
+        Published: <%= @entry.inserted_at |> EChronicler.Models.JournalEntry.format_datetime() %> 
         </div>
+        <%= if @entry.inserted_at != @entry.updated_at do %>
+        Updated: <%= @entry.updated_at |> EChronicler.Models.JournalEntry.format_datetime() %> 
+        <% end %>
         <hr id="title-body-divider">
         <p id="entry-text"> <%= @entry.entry %> </p>
       </article>
       <hr id="dotted-post-divider">
+
+      <button style="color: white;"><%= link "Edit", to: Routes.journal_entry_path(@conn, :edit, @entry ) %>
```

Comment:
I pulled down the branch and it worked great! A small feature request if you have time (I know it's not in the acceptance criteria): can you add a button here to go back to the home page?

Reply:
Done!